cargo-shuttle/tests/integration/run.rs

```diff
@@ -359,3 +373,19 @@ async fn thruster_hello_world() {
 
     assert_eq!(request_text, "Hello, World!");
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn rocket_hello_world_with_router_ip() {
+    let port = cargo_shuttle_run("../examples/rocket/hello-world", true).await;
+
+    let request_text = reqwest::Client::new()
+        .get(format!("http://0.0.0.0:{port}/hello"))
```

Comment:
Reason for returning the url (or full address) from the helper function.

Reply:
Amended - please see new commit